.eslintignore

```diff
@@ -0,0 +1 @@
+test/
```

Comment:
why no linting for tests ?

Reply:
had some generated code there, I'll move things around in an upcoming PR

Reply:
thanks